### PR TITLE
docs: Update automator-sdk documentation

### PR DESCRIPTION
**Pull request by andrei-bitca-dc**
- Remove the need of meridian flag
- Add documentation for Azure Devops Extensions 
- Add documentation for automator statistics
- Add documentation for integrations
- Add documentation for global templates

### CODE REVIEW

File: docs/integrations.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+In order to use this feature, an additional header needs to be passed:
+
+- `Accept: application/vnd.deepcrawl.tray-io-integration-preview` 
+
+## Getting an integration by id
+
+```graphql
+{
+  node(id: "TjAxMUludGVncmF0aW9uNg") {
```
Comment (izy-izzy) [inline docs/integrations.md:12]: Should we put something like this here ? 
`node(id: "<string>") {`

File: docs/test-suites.md [hunk 2]
```diff
@@ -280,4 +281,53 @@ Name | Type
 `useRenderer` | Boolean
 `useRobotsOverwrite` | Boolean
 
-<!-- tabs:end -->
\ No newline at end of file
+<!-- tabs:end -->
+
+## Global Templates
+
+This feature enables you to link two or multiple test suites together, by making one of it a global template. When any of the global template configuration in Step 2 or any test configuration in Step 3 is updated, all the linked test suites configurations and test configurations will be updated.
```
Comment (izy-izzy) [inline docs/test-suites.md:288]: Please mention what is not copied. We have some exceptions.

File: docs/integrations.md (hunk 1 above)
Comment (andrei-bitca-dc) [reply docs/integrations.md:12]: the whole documentation is based on examples, I tried to keep it compliant by providing an example id. if we want to do it this way, we need to modify it everywhere. let me know.

File: docs/test-suites.md (hunk 2 above)
Comment (andrei-bitca-dc) [reply docs/test-suites.md:288]: hmm, to be honest, I don't remember the exceptions, that's why I didn't mention them

File: docs/azure-devops-extension.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+It has the following tasks:
+
+## Run Automator Build Task (`run-automator-build-task`)
+
+### How to run
+
+- Ask for extension permissions
+- Once permission is given, install the extension from: https://marketplace.visualstudio.com/items?itemName=deepcrawl.automator-tools
+- Generate an API key on: https://app.deepcrawl.com/dc-api
+- Make sure you have the test suite created and the id, if not create your test suite in automator: https://automator.deepcrawl.com/
```
[the remark this answers is not in the document]
Comment (andrei-bitca-dc) [reply docs/azure-devops-extension.md:14]: needs update in multiple places

File: docs/integrations.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
+# Integrations
+---
+
+In order to use this feature, an additional header needs to be passed:
+
+- `Accept: application/vnd.deepcrawl.tray-io-integration-preview` 
+
+## Getting an integration by id
+
```
[the remark this answers is not in the document]
Comment (andrei-bitca-dc) [reply docs/integrations.md:9]: @ayemonchit the only reason I skipped JIRA is that: there's an abstraction behind it and some queries / mutations can be used for all kinds of integrations that will be added in the future. 

An integration can be of multiple types.

[the remark this answers is not in the document]
Comment (andrei-bitca-dc) [reply docs/integrations.md:9]: I think I can do the JIRA integrations for now though, as it's the only one we have implemented so far.

File: docs/test-suites.md (hunk 2 above)
Comment (izy-izzy) [reply docs/test-suites.md:288]: crawl type is always set to web.  ( so not file upload also)
startURL is also not copied.

**Review by ayemonchit — APPROVED** [review]

**Review by izy-izzy — APPROVED** [review]
Nice

**Pull request merged**